### ISSUE TITLE
Type '(config: Config) => Config' is not assignable to type 'Plugin'.

### ISSUE THREAD

**Issue by ccfiel** [boomworks/payload-plugin-author-fields#18]
I tried to install your plugin, but I got this error. Any idea why? I am using payload 2.0.0
```
Type '(config: Config) => Config' is not assignable to type 'Plugin'.
```
<img width="1001" alt="SCR-20231021-mtfh" src="https://github.com/boomworks/payload-plugin-author-fields/assets/1788360/204305a3-d385-4c67-9a82-00aa99a47f01">

**Comment by lindsayevans** [comment]
Hi @ccfiel - I've just published v1.1.4 of the plugin with updated peerDependencies - can you give that a try?

**Comment by ccfiel** [comment]
@lindsayevans Thanks tried to update but I got another error

```
[00:20:47] ERROR (payload): ReferenceError: window is not defined
    at Object.condition (/Users/chrisianfiel/Projects/lten/node_modules/@boomworks/payload-plugin-author-fields/src/addAuthorFields.ts:160:24)
    at promise (/Users/chrisianfiel/Projects/lten/node_modules/payload/src/fields/hooks/beforeChange/promise.ts:55:19)
    at forEach (/Users/chrisianfiel/Projects/lten/node_modules/payload/src/fields/hooks/beforeChange/traverseFields.ts:46:14)
    at Array.forEach (<anonymous>)
    at traverseFields (/Users/chrisianfiel/Projects/lten/node_modules/payload/src/fields/hooks/beforeChange/traverseFields.ts:44:10)
    at beforeChange (/Users/chrisianfiel/Projects/lten/node_modules/payload/src/fields/hooks/beforeChange/index.ts:37:23)
    at updateByID (/Users/chrisianfiel/Projects/lten/node_modules/payload/src/collections/operations/updateByID.ts:224:36)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at async resetLoginAttempts (/Users/chrisianfiel/Projects/lten/node_modules/payload/src/auth/strategies/local/resetLoginAttempts.ts:18:3)
    at async unlock (/Users/chrisianfiel/Projects/lten/node_modules/payload/src/auth/operations/unlock.ts:64:7)
```

same with this issue https://github.com/boomworks/payload-plugin-author-fields/issues/19
